tests/components/home_connect/test_config_flow.py

```diff
@@ -60,7 +60,7 @@ async def test_full_flow(
 
     client = await hass_client_no_auth()
     resp = await client.get(f"/auth/external/callback?code=abcd&state={state}")
-    assert resp.status == HTTPStatus.OK
+    assert resp.status is HTTPStatus.OK
```

Comment:
We can't use identify check here I believe. The status attribute is an integer, not an instance of `HTTPStatus`.